.github/workflows/build.yml

```
@@ -414,6 +414,35 @@ jobs:
       - name: Build and test
         run: ./Android/android.py ci --fast-ci ${{ matrix.arch }}-linux-android
 
+  build-ios:
+    name: iOS (${{ matrix.arch }})
```

We include the `arch` for Android to differentiate the two, so can skip it here:
```suggestion
    name: iOS
```